ezcontent_publish.module

```diff
@@ -102,20 +102,20 @@ function ezcontent_publish_form_shortcut_set_customize_form_alter(&$form, $form_
  */
 function ezcontent_publish_form_alter(&$form, $form_state, $form_id) {
   $currentUser = \Drupal::currentUser();
-  if (!in_array('administrator', $currentUser->getRoles())) {
-    $form['shortcuts']['links']['#empty'] = t('No shortcuts available. <a href=":link">Add a shortcut</a>', [':link' => Url::fromRoute('ezcontent_publish.shortcut.link_add', ['shortcut_set' => 'shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id()])->toString()]);
-    if ($form_id === 'shortcut_confirm_delete' || $form_id === 'shortcut_shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id() . '_form' || $form_id === 'shortcut_shortcut_set_' . $currentUser->getUsername() . '_'  . $currentUser->id() . '_add_form') {
-      $form['actions']['submit']['#submit'][] = 'ezcontent_publish_shortcut_form_submit';
-    }
+  $form['shortcuts']['links']['#empty'] = t('No shortcuts available. <a href=":link">Add a shortcut</a>', [':link' => Url::fromRoute('ezcontent_publish.shortcut.link_add', ['shortcut_set' => 'shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id()])->toString()]);
+  if ($form_id === 'shortcut_default_add_form' || $form_id === 'shortcut_set_customize_form' || $form_id === 'shortcut_default_form' || $form_id === 'shortcut_confirm_delete' || $form_id === 'shortcut_shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id() . '_form' || $form_id === 'shortcut_shortcut_set_' . $currentUser->getUsername() . '_'  . $currentUser->id() . '_add_form') {
+    $form['actions']['submit']['#submit'][] = 'ezcontent_publish_shortcut_form_submit';
   }
 }
 
 /**
  * Implements ezcontent_publish_shortcut_form_submit().
  */
-function ezcontent_publish_shortcut_form_submit(array $form, $form_state){
-  Cache::invalidateTags(['ezcontent_user_shortcut_block']);
-  $shortcutSet = $form_state->getBuildInfo()['callback_object']->getEntity()->get('shortcut_set')->getValue()[0]['target_id'];
-  $redirectUrl = Url::fromRoute('ezcontent_publish.shortcut_set.customize_form', array('shortcut_set' => $shortcutSet));
-  $form_state->setRedirectUrl($redirectUrl);
+function ezcontent_publish_shortcut_form_submit(array $form, $form_state) {
+  $currentUser = \Drupal::currentUser();
+  if (!in_array('administrator', $currentUser->getRoles())) {
```

Comment:
Since this is the only place where `$currentUser` is being used, let's avoid the variable declaration and directly use: `\Drupal::currentUser()->getRoles()`